source/docs/advanced/production-ready/logging.md

```diff
@@ -1,17 +1,18 @@
 ---
 id: logging
 title: Setup and use logging
+description: Front-Commerce uses the winston library for logging. This guide explains how to configure it and use it in your own code.
 ---
 
-When you need to log something in a javascript environment, the first thing that comes to mind is obviously `console.log`. But the issue with it is that you won't be able access it in the long term. Especially since some Node.js server environments don't keep a trace of the stdout/stderr of your application.
+When you need to log something in a javascript environment, the first thing that comes to mind is `console.log`. But the issue with it is that you won't be able access it in the long term. Especially since some Node.js server environments don't keep a trace of the stdout/stderr of your application.
```

Comment:
```suggestion
When you need to log something in a javascript environment, the first thing that comes to mind is `console.log`. But the issue with it is that you won't be able to access it in the long term. Especially since some Node.js server environments don't keep a trace of the stdout/stderr of your application.
```